css/carouselImages.css

```diff
@@ -0,0 +1,191 @@
+:root {
+  --primary-color: #ff6600;
+  --primary-hover-color: #cc5200;
+  --main-color: #141414;
+  --secondary-hover-color: #e0e0e0;
+  --background-color: #f8f8f8;
+  --light-color: #ffffff;
+  --shadow-color: rgba(0, 0, 0, 0.02);
+  --button-hover-color: #e6e6e6;
```

Comment:
**CSS Custom Properties (Variables) Organization:**
   - Group and comment related variables for better readability and maintainability. For example, group all color-related variables together, spacing, etc.

   ```css
   :root {
     /* Colors */
     --primary-color: #ff6600;
     --primary-hover-color: #cc5200;
     --main-color: #141414;
     --secondary-hover-color: #e0e0e0;
     --background-color: #f8f8f8;
     --light-color: #ffffff;
     --button-hover-color: #e6e6e6;

     /* Shadows */
     --shadow-color: rgba(0, 0, 0, 0.02);

     /* Spacing (Example) */
     --space-lg: 7.5rem; /* 120px */
     --space-md: 1.25rem; /* 20px */
   }
   ```